BusinessCore.Tests/alloretTestsFunctional.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BusinessCore.Tests.Services;
+using CarManagement.Core.Models;
+using CarManagement.Core.Services;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    public class alloretTestsFunctional
+    {
+        private readonly alloretArrayVehicleStorage vehicleStorage;
+
+        public alloretTestsFunctional()
+        {
+            this.vehicleStorage = new alloretArrayVehicleStorage();
+        }
+
+        [TestMethod]
+        public void minimal_horsepower_of_pressure_value_wheel_3_and_started_engine_cars_is_85cv()
```

Comment:
please, rename to a more comprehensive name or sentence like **minimal_horsePower_for_vehicles_with_wheel_that_have_three_atmospheres_of_pressure_is_85cv**.